api/src/main/java/io/opentelemetry/OpenTelemetry.java

```diff
@@ -5,264 +5,253 @@
+  }
+
+  /** Returns the global {@link BaggageManager}. */
+  static BaggageManager getGlobalBaggageManager() {
+    return getGlobalOpenTelemetry().getBaggageManager();
   }
 
   /**
-   * Returns a singleton {@link BaggageManager}.
-   *
-   * @return registered manager or default via {@link DefaultBaggageManager#getInstance()}.
-   * @throws IllegalStateException if a specified manager (via system properties) could not be
-   *     found.
-   * @since 0.1.0
+   * Returns the globally registered {@link ContextPropagators} for remote propagation of a context.
    */
-  public static BaggageManager getBaggageManager() {
-    return getInstance().contextManager;
+  static ContextPropagators getGlobalPropagators() {
+    return getGlobalOpenTelemetry().getPropagators();
   }
 
   /**
-   * Returns a {@link ContextPropagators} object, which can be used to access the set of registered
-   * propagators for each supported format.
-   *
-   * @return registered propagators container, defaulting to a {@link ContextPropagators} object
-   *     with {@link HttpTraceContext} registered.
-   * @throws IllegalStateException if a specified manager (via system properties) could not be
-   *     found.
-   * @since 0.3.0
+   * Sets the globally registered {@link ContextPropagators} for remote propagation of a context.
    */
-  public static ContextPropagators getPropagators() {
-    return getInstance().propagators;
+  static void setGlobalPropagators(ContextPropagators propagators) {
+    requireNonNull(propagators, "propagators");
+    setGlobalOpenTelemetry(
+        getGlobalOpenTelemetry().toBuilder().setPropagators(propagators).build());
   }
 
+  /** Returns the {@link TracerProvider} for this {@link OpenTelemetry}. */
+  TracerProvider getTracerProvider();
+
   /**
-   * Sets the {@link ContextPropagators} object, which can be used to access the set of registered
-   * propagators for each supported format.
+   * Gets or creates a named tracer instance from the {@link TracerProvider} for this {@link
+   * OpenTelemetry}.
+   *
+   * <p>This is a shortcut method for {@code getTracerProvider().get(instrumentationName)}.
    *
-   * @param propagators the {@link ContextPropagators} object to be registered.
-   * @throws IllegalStateException if a specified manager (via system properties) could not be
-   *     found.
-   * @throws NullPointerException if {@code propagators} is {@code null}.
-   * @since 0.3.0
+   * @param instrumentationName The name of the instrumentation library, not the name of the
+   *     instrument*ed* library (e.g., "io.opentelemetry.contrib.mongodb"). Must not be null.
+   * @return a tracer instance.
    */
-  public static void setPropagators(ContextPropagators propagators) {
-    Objects.requireNonNull(propagators, "propagators");
-    getInstance().propagators = propagators;
+  default Tracer getTracer(String instrumentationName) {
+    return getTracerProvider().get(instrumentationName);
```

Comment:
I'm a bit worried about this. The docs say it is just a shortcut, but it is overridable.

Reply:
Great point, removed the wording about shortcuts for the interface methods since not a big deal.

Reply:
I would not be sure if this is not a big deal. This now allows creation of a tracer that does not go through a TracerProvider. I think we must at least add javadoc that this method should not be overridden, or if it is, it must only create a tracer through a TracerProvider.

Reply:
If someone wants to muck about with the API in non-spec-compliant ways, isn't that really their problem? Not sure why this matters all that much in the real world.